pipelines/optimus/README.md

```diff
@@ -193,23 +197,21 @@ Output files of the pipeline include:
 3. Cell metadata, including cell metrics
 4. Gene metadata, including gene metrics
 
-The following table lists the output files produced from the pipeline. For samples that have sequenced over multiple lanes, the pipeline will output one merged version of each listed file.
+The following table lists the output files produced from the pipeline. For samples that have sequenced over multiple lanes, the pipeline will output one merged version of each listed file. 
 
 | Output Name | Filename, if applicable | Output Type |Output Format |
 | ------ |------ | ------ | ------ | 
 | pipeline_version | | Version of the processing pipeline run on this data | String | 
-| bam | merged.bam | aligned bam | bam |
+| bam | sample_id.bam | Aligned BAM | BAM |
 | matrix_row_index | sparse_counts_row_index.npy | Index of cells in expression matrix | Numpy array index |
 | matrix_col_index | sparse_counts_col_index.npy | Index of genes in expression matrix | Numpy array index | 
 | cell_metrics | merged-cell-metrics.csv.gz | cell metrics | compressed csv | Matrix of metrics by cells | 
 | gene_metrics | merged-gene-metrics.csv.gz | gene metrics | compressed csv | Matrix of metrics by genes | 
 | loom_output_file | output.loom | Loom | Loom | Loom file with expression data and metadata | N/A |
 
+The BAM output has the default prefix "sample_id", but this is an optional parameter; the BAM file can be renamed in the [Optimus WDL](Optimus.wdl) by replacing the "sample_id" in the code line `output_bam_filename = sample_id + ".bam"`. 
```

Comment:
I think we should include this as an optional input so users can specify an output base name in the input json rather than having to change the wdl itsef

Reply:
Sorry, hit the wrong button and resolved but I’ve fixed it. 

Just here to second this and say it’s what we talked about in refinement and was commented on the PR. Can this be fixed before we release?

Reply:
Yes, this is a very simple code change